client/src/components/Ratings/sub-components/UploadImageModal.jsx

```diff
@@ -1,120 +1,144 @@
-// /// REVISIT LATER ///////
-// /// DEPENDENT ON AN EXTERNAL API -CAUSING LOCALISSUES /////
+/// REVISIT LATER ///////
+/// DEPENDENT ON AN EXTERNAL API -CAUSING LOCALISSUES /////
 
-// import React, { useState } from 'react';
-// import ReactDOM from 'react-dom';
-// import axios from 'axios';
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+import { FcUpload } from 'react-icons/fc';
 
-// export default function UploadAndDisplayImage({
-//   setImageUploadModal,
-//   selectedImage,
-//   setSelectedImage,
-//   submitReview,
-//   setSubmitReview,
-// }) {
-//   // STATE DATA //
-//   const [imgProgress, setImgProgress] = useState({
-//     0: false, 1: false, 2: false, 3: false, 4: false,
-//   });
+export default function UploadAndDisplayImage({
+  setImageUploadModal,
+  selectedImage,
+  setSelectedImage,
+  newReview,
+  setNewReview,
+  imgProgress,
+  setImgProgress,
+}) {
+  // STATE DATA //
+  const [numUploaded, setNumUploaded] = useState(0);
 
-//   // HELPER FUNCTIONS //
-//   const imageURLGenerator = (imgPath) => {
-//     const options = {
-//       url: 'http://localhost:3000/api/image',
-//       method: 'post',
-//       data: { imgPath },
-//     };
-//     axios(options)
-//       .then(({ data }) => setSubmitReview({
-//         ...submitReview,
-//         photos: [...submitReview.photos, data],
-//       }))
-//       .catch((err) => console.error(err));
-//   };
+  // HELPER FUNCTIONS //
+  const imageURLGenerator = (imgPath) => {
```

Comment:
I will get with you on this to see if we can use the same process for uploading photos. In the future, ideally, we would only have one shared modal that is the photo upload dialogue.